spec/models/movie_sync_spec.rb

```diff
@@ -3,18 +3,18 @@
 RSpec.describe MovieSync do
   describe "#run" do
     it "fetches the top 100 movies from TMDB and inserts them into the DB" do
-      fight_club = build(
-        :movie,
+      fight_club = {
+        tmdb_id: 1,
```

Comment:
Would it be useful to use factorybot's `attributes_for` here?

```suggestion
      fight_club = attributes_for(
        :movie,
```
This will autogenerate the `tmdb_id` for us, and also fill in the rest of the attributes from their defaults.

Reply:
Oh, nice!